script/bootstrap

```diff
@@ -14,6 +14,14 @@ if [ -z "$GOPATH" ]; then
     ln -s "$GOPATH" src/github.com/git-lfs/git-lfs
 fi
 
+if uname -s | grep -q "_NT-"; then
+  echo "Installing goversioninfo to embed resources into Windows executables..."
```

Comment:
I believe `go get` is smart enough to do something like that itself. Here's some timing:

    $ time go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo

    real    0m4.039s
    user    0m0.015s
    sys     0m0.000s

    $ time go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo

    real    0m0.245s
    user    0m0.000s
    sys     0m0.000s

As you can see, the second time `go get` takes only 0.2s.

Anyway, I was considering to use Glide for `goversioninfo` and commit it under `vendor`, but when I tried quickly I wasn't able to get it working correctly. Would be using Glide be the better approach, even though `goversioninfo` is only used for the Windows build?